src/main/java/com/iexec/resultproxy/jwt/JwtService.java

```diff
@@ -54,11 +54,7 @@ public boolean isValidJwt(String jwtString) {
             return false;
         }
 
-        if (jwtString.equals(oExistingJwt.get().getJwtString())) {
-            return true;
-        }
-
-        return false;
+        return jwtString.equals(oExistingJwt.get().getJwtString());
```

Comment:
```suggestion
        return oExistingJwt.isPresent() && jwtString.equals(oExistingJwt.get().getJwtString());
```
This could allow to remove the previous if block as well
What do you think ?

Reply:
You're right, thanks! That's fixed in https://github.com/iExecBlockchainComputing/iexec-result-proxy/pull/46/commits/8e0a91cfefec74beaecf6bd8de4ff7af204e93a2.